### PR TITLE
[9.0] [ADD] Normalize accented chars. in product search

### PR DESCRIPTION
**Pull request by Fkawala**
This PR provides normalization for the POS product search string. The normalization allows user to type indistinctively "café" or "cafe". The normalization applies to all the Diacritical marks in range \u0300-\u036f and œ.

### CODE REVIEW

File: pos_accented_search/__openerp__.py
Comment (hbrunn) [inline pos_accented_search/__openerp__.py]: this should be a user friendly name, the technical name is already the module name

File: pos_accented_search/__openerp__.py
Comment (hbrunn) [inline pos_accented_search/__openerp__.py]: this should go to `README.rst`

File: pos_accented_search/__openerp__.py
Comment (hbrunn) [inline pos_accented_search/__openerp__.py]: this also needs to contain the OCA: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#modules (you should read and follow all of this, otherwise review becomes tedious. I'm going to ignore everything mentioned in the conventions from now on)

**Review by hbrunn — CHANGES_REQUESTED** [review]
please also check https://travis-ci.org/OCA/pos/jobs/639692786#L689-L713

File: pos_accented_search/__openerp__.py
Comment (Fkawala) [reply pos_accented_search/__openerp__.py]: Will do that thank you.

Comment (robinkeunen) [reply pos_accented_search/__openerp__.py]: Don't hesitate to reach me for assistance.

File: pos_accented_search/static/src/js/db.js
Comment (robinkeunen) [inline pos_accented_search/static/src/js/db.js]: I'd call this `normalize_characters` as you're nit actually removing them.

File: pos_accented_search/__openerp__.py
Comment (Fkawala) [reply pos_accented_search/__openerp__.py]: Thank you, I'll give it a try and call for help if needed!

**Review by robinkeunen — CHANGES_REQUESTED** [review]
The code LGTM, thank you for the contribution.  As hbrunn suggested, there are some OCA conventions to take into account. Don't hesitate to reach me for assistance.

File: pos_accented_search/static/src/js/db.js
Comment (Fkawala) [reply pos_accented_search/static/src/js/db.js]: done.

File: pos_accented_search/README.rst
Comment (hbrunn) [inline pos_accented_search/README.rst]: makeS

**Review by hbrunn — APPROVED** [review]
thanks, just one last typo

File: pos_accented_search/README.rst
Comment (Fkawala) [reply pos_accented_search/README.rst]: Arg sorry about that.

**Review by robinkeunen — APPROVED** [review]

**Comment by Fkawala** [comment]
Hello @robinkeunen what is the next step to get that merged?

**Comment by robinkeunen** [comment]
Wait for the maintainers to merge the request. We can already deploy it on your servers though. I'll see with the team if we can do that this week.

**Comment by Fkawala** [comment]
@robinkeunen no need to rush and do extra work for this as we already have a working solution in prod, thank you.

**Pull request merged**
